WalletWasabi.Fluent/Helpers/NotificationHelpers.cs

```diff
@@ -68,23 +71,24 @@ private static bool TryGetNotificationInputs(ProcessedResult result, [NotNullWhe
 				Money incoming = receivedSum - spentSum;
 				Money receiveSpentDiff = incoming.Abs();
 				string amountString = receiveSpentDiff.ToFormattedString();
+				string fiatString = receiveSpentDiff.ToUsd(wallet.Synchronizer.UsdExchangeRate).ToUsdAmount();
 
 				if (result.Transaction.Transaction.IsCoinBase)
 				{
-					message = $"{amountString} BTC received as Coinbase reward";
+					message = $"{amountString} BTC ({fiatString} USD) received as Coinbase reward";
 				}
 				else if (isSpent && receiveSpentDiff == miningFee)
 				{
 					message = $"Self transfer";
 				}
 				else if (incoming > Money.Zero)
 				{
-					message = $"{amountString} BTC incoming";
+					message = $"{amountString} BTC ({fiatString} USD) incoming";
 				}
 				else if (incoming < Money.Zero)
 				{
 					var sentAmount = receiveSpentDiff - miningFee;
-					message = $"{sentAmount.ToFormattedString()} BTC sent";
+					message = $"{sentAmount.ToFormattedString()} BTC ({fiatString} USD) sent";
```

Comment:
Here the `fiatString` contains also the transaction fee. But the BTC amount does not. Fiat mount should not contain the transaction fee either.